ext/RMagick/rmstruct.c

```diff
@@ -605,7 +605,7 @@ VALUE
 Import_PrimaryInfo(PrimaryInfo *p)
 {
     return rb_funcall(Class_Primary, rm_ID_new, 3,
-                      INT2FIX(p->x), INT2FIX(p->y), INT2FIX(p->z));
```

Comment:
Is this the right move? Should this be a float? [The docs for `Chromaticity`](https://rmagick.github.io/struct.html#Chromaticity) show floating point numbers as examples, and the default values are floating point.